charts/internal/gcp-infra/values.yaml

```diff
@@ -16,6 +16,9 @@ clusterName: test-namespace
 networks:
   cloudNAT:
     minPortsPerVM: 2048
+#   natIPs: ["${data.google_compute_address.manualnat1.self_link}","${data.google_compute_address.manualnat2.self_link}"]
+#   natIPNames: ["manualnat1", "manualnat2"]
+#   natIPOutput:   "${data.google_compute_address.manualnat1.address},${data.google_compute_address.manualnat2.address}"
```

Comment:
Wait a sec. Why does a user has to specify this? Same question for `natIPs`.
Imo, `natIPNames` is the only value that should be specified by a user. The other values can be computed on the fly and are an implementation detail, or?

Reply:
They are calculated on the fly, will remove the comments.